create_release.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
+
+echo "Add config for alphagov/govuk_elements"
+
+git config --global user.name "Travis CI"
+git config --global user.email "travis@travis-ci.org"
+git remote add origin_ssh git@github.com:alphagov/govuk_elements.git
+
+# This openssl command was generated automatically by `travis encrypt-file`, see `.travis/README.md` for more details
+openssl aes-256-cbc -K $encrypted_85ebe8034b89_key -iv $encrypted_85ebe8034b89_iv -in .travis/govuk_elements.enc -out ~/.ssh/id_rsa -d && chmod 600 ~/.ssh/id_rsa
+
+echo "Check to see if the version file has been updated"
+
+# Get the version from the version file
+VERSION_TAG="v`cat VERSION.txt`"
+
+# Create a new tag - if the version file has been updated and a tag for that
+# version doesn't already exist
+
+# Check to make sure the tag doesn't already exist
+if ! git rev-parse $VERSION_TAG >/dev/null 2>&1; then
+  echo "Creating new tag: $VERSION_TAG"
+
+  # Create a new tag and push to Github
```

Comment:
Worth a comment to say that pushing the tag triggers other builds? If someone is starting from here?